src/components/error-summary/linking-checkboxes-radios/index.njk

```diff
@@ -39,15 +39,17 @@ layout: layout-example.njk
       hint: {
         text: "including English, Scottish, Welsh and Northern Irish"
       },
-      id: "nationality"
+      id: "nationality-british"
     },
     {
       value: "irish",
-      text: "Irish"
+      text: "Irish",
+      id: "nationality-irish"
     },
     {
       value: "other",
-      text: "Citizen of another country"
+      text: "Citizen of another country",
+      id: "nationality-other"
```

Comment:
What about leaving all of the `id`s out, and linking to `#nationality`, relying on the automatic ID of the first input?

I think that is more representative of what a user would normally do, rather than assigning each input an ID.